dotcom-rendering/src/web/lib/cardWrappers.tsx

```diff
@@ -223,6 +223,42 @@ export const Card25Media25 = ({
 		/>
 	);
 };
+/**
+ * ┏━━━━━━━━━┱┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┐
+ * ┃▒▒▒▒▒▒▒▒▒┃            75%            ┊
+ * ┃▒▒▒▒▒▒▒▒▒┃         Remaining         ┊
+ * ┃         ┃                           ┊
+ * ┗━━━━━━━━━┹┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┘
+ * Card designed to take up 25% of the container, with media that takes up 25%
+ *
+ * Options:
+ *  - Medium headline (medium on mobile)
+ *  - Small image on the top (left on mobile)
+ *  - No trail text
+ *  - Up to 2 supporting content items, always aligned vertical
+ *  - Added star rating
+ */
+export const Card25Media25StarRating = ({
```

Comment:
In hindsight, kind of wondering if the variants are needed - from the [discussion](https://github.com/guardian/dotcom-rendering/pull/5605) when star ratings were added to DCR, it sounds as though the preferred design would be to display ratings if they're available - so I guess the option could just be added to the standard Card25Media25 wrapper? 

However it doesn't look like all containers in DCR were using this property, so it felt appropriate to create a variant.